local.properties

```diff
@@ -0,0 +1,10 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Fri Jun 25 14:18:43 IST 2021
+sdk.dir=/Users/reytum/Library/Android/sdk
```

Comment:
is this required?

Reply:
No, But I committed the file because it had entries for jfrog. This path will be changed locally when someone clones the SDK.